_posts/2019-04-16-metals.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: blog-detail
+post-type: blog
+by: Ólafur Páll Geirsson, Gabriele Petronella, Jorge Vicente Cantero
+title: "Write Scala in VS Code, Vim, Emacs, Atom and Sublime Text with Metals"
+---
+
+We are excited to announce the release of
+[Metals v0.5](https://scalameta.org/metals/blog/2020/04/12/mercury.html). Metals
+is a Scala language server that supports code completions, type at point, goto
+definition, fuzzy symbol search and other advanced code editing and navigation
+capabilities. Metals can be used in VS Code, Vim, Emacs, Atom and Sublime Text
+as well as any other
+[Language Server Protocol](https://microsoft.github.io/language-server-protocol/)
+compatible editor. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) along with contributors from the
+community.
+
+## Features
+
+This post we are going to demonstrate how to use Metals with VS Code. To get
+started, install the
+[Scala (Metals)](https://marketplace.visualstudio.com/items?itemName=scalameta.metals)
+extension on the VS Code Marketplace and open an sbt project directory. The
+Metals extension will prompt you to import the build.
+
+![Import sbt build](https://i.imgur.com/0VqZWay.png)
```

Comment:
Should we store the images on our own servers instead?

Reply:
I kept the external links since I wanted to avoid bloating the git repository with GIFs. I can download and embed the images, wdyt @SethTisue ?

Reply:
I don't have a strong opinion either way. We don't publish that many blog posts, so it's probably okay.

Reply:
I kept the external URLs. In general, I prefer external links because it makes the markdown source more portable. With local image links you can't as easily share the markdown source.